packages/core/test/financial-templates/common/FundingRateApplier.js

```diff
@@ -196,11 +196,32 @@ contract("FundingRateApplier", function(accounts) {
     assert.equal(test4[1].rawValue, toWei("0"));
   });
 
-  it("Initial 0 funding rate", async () => {
+  it("Initial funding rate struct is correct", async () => {
```

Comment:
This test is important because the `updateTime` and `applicationTime` are set on construction to the deployment time, we should explicitly check for this.